src/builders/fileSystem.ts

```diff
@@ -6,11 +6,11 @@ import { Logger } from "../options/logger";
 import { SemicolonCharacter } from "../options/noSemicolon";
 import { QuoteCharacter } from "../options/quoteCharacter";
 import {
-  Directory,
   indentation,
-  Location,
   nonAlphaNumeric,
 } from "../utilities";
+import { Directory } from "../interfaces/directory.interface";
+import { FileTreeLocation } from "../interfaces/location.interface";
```

Comment:
Looks like there's still some remaining doublequotes in places - could be worth running prettier on all files to get to a good baseline if we're introducing it now